security/s2a/internal/crypter/counter.go

```diff
@@ -0,0 +1,42 @@
+package crypter
+
+import "errors"
+
+const counterLen = 12
```

Comment:
Since we need to use a 64-bit sequence number, I think this should be 8.

Reply:
Done